releases/release-5.1.2.md

```diff
@@ -0,0 +1,153 @@
+
++ TiKV
+
+    - Fix a bug that some files are missed during the process of importing snapshot files when upgrading TiKV from v3.x to later versions [#10902](https://github.com/tikv/tikv/issues/10902)
+    - Fix the potential disk full issue caused by corrupted snapshot files [#10813](https://github.com/tikv/tikv/issues/10813)
+    - Make the slow log of TiKV coprocessor only consider the time spent on processing requests [#10841](https://github.com/tikv/tikv/issues/10841)
+    - Drop log instead of blocking threads when the slogger thread is overloaded and the queue is filled up [#10841](https://github.com/tikv/tikv/issues/10841)
+    - Fix a panic issue that occurs when processing Coprocessor requests times out [#10852](https://github.com/tikv/tikv/issues/10852)
+    - Fix the TiKV panic issue that occurs when upgrading from a pre-5.0 version with Titan enabled [#10842](https://github.com/tikv/tikv/pull/10842)
+    - Fix the issue that TiKV of a newer version cannot be rolled back to v5.0.x [#10842](https://github.com/tikv/tikv/pull/10842)
+    - Fix the issue that TiKV might delete files before it ingests data to RocksDB [#10438](https://github.com/tikv/tikv/issues/10438)
+    - Fix the parsing failure caused by the left pessimistic locks [#26404](https://github.com/pingcap/tidb/issues/26404)
+
++ PD
+
+    - Fix the issue that PD does not fix the down peers in time [#4077](https://github.com/tikv/pd/issues/4077)
+    - Fix the issue that the replica count of the default placement rules stays constant after `replication.max-replicas` is updated [#3886](https://github.com/tikv/pd/issues/3886)
+    - Fix a bug that PD might panic when scaling out TiKV [#3868](https://github.com/tikv/pd/issues/3868)
+    - Fix a bug that the hot Region scheduler cannot work when the cluster has the evict leader scheduler [#3697](https://github.com/tikv/pd/issues/3697)
+
++ TiFlash
+
+    - Fix the issue of unexpected results when TiFlash fails to establish MPP connections
+    - Fix the potential issue of data inconsistency that occurs when TiFlash is deployed on multiple disks
+    - Fix a bug that MPP queries get wrong results when TiFlash server is under high load
+    - Fix a potential bug that MPP queries hang forever
+    - Fix the issue of concurrency issue between store initialization and DDL
+    - Fix a bug of incorrect results that occurs when queries contain filters like `CONSTANT`, `<`, `<=`, `>`, `>=` or `COLUMN`
+    - Fix the potential panic issue when `Snapshot` is applied simultaneously with multiple DDL operations
+    - Fix the issue that the store size in metrics is inaccurate under heavy writing
+    - Fix the potential issue that TiFlash cannot perform GC for the delta data after running for a long time
+    - Fix the issue of wrong results when new collation is enabled
+    - Fix the potential panic issue that occurs during lock resolving
+    - Fix a bug that metrics display wrong value
```

Comment:
- Line 112: personal suggestion refer to line 124
- Line 123: [5.0.4](https://github.com/pingcap/docs/pull/6449/files), line 165
- Line 126: [5.0.4](https://github.com/pingcap/docs/pull/6449/files), line 166
- Line 127: personal suggestion 
- Line 128: [5.0.4](https://github.com/pingcap/docs/pull/6449/files), line 163
- Line 129: [5.0.4](https://github.com/pingcap/docs/pull/6449/files), line 167


```suggestion
+ TiFlash

    - Fix the issue of unexpected results when TiFlash fails to establish MPP connections
    - Fix the potential issue of data inconsistency that occurs when TiFlash is deployed on multiple disks
    - Fix a bug that MPP queries get wrong results when TiFlash server is under high load
    - Fix a potential bug that MPP queries hang forever
    - Fix the panic issue when operating store initialization and DDL simultaneously
    - Fix a bug of incorrect results that occurs when queries contain filters like `CONSTANT`, `<`, `<=`, `>`, `>=`, or `COLUMN`
    - Fix the potential panic issue when `Snapshot` is applied simultaneously with multiple DDL operations
    - Fix the issue that the store size in metrics is inaccurate under heavy writing
    - Fix the potential issue that TiFlash cannot garbage-collect the delta data after running for a long time
    - Fix the issue of wrong results when the new collation is enabled
    - Fix the potential panic issue that occurs when resolving locks
    - Fix a potential bug that metrics display wrong values
```

Reply:
1. Modification in lines 127 and 133 may result in dangling structure, which means the intended subject differs from the subject of the sentence.
2. Maybe GC is a common word that does not need the full name.